process/track/baseBlockTrack.go

```diff
@@ -78,6 +92,20 @@ func (bbt *baseBlockTrack) receivedMetaBlock(headerHandler data.HeaderHandler, m
 		return
 	}
 
+	err := bbt.checkBlockBasicValidity(metaBlock)
```

Comment:
no need for this as implementation is nil. log.Trace will be never called, neither return.

Reply:
function removed